packages/big-design/src/components/Worksheet/editors/ModalEditor/styled.ts

```diff
@@ -0,0 +1,22 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+import { StyleableButton } from '../../../Button/Button';
+
+export const StyledButton = styled(StyleableButton)`
+  font-size: ${({ theme }) => theme.typography.fontSize.small};
+  height: auto;
+  line-height: ${({ theme }) => theme.lineHeight.small};
+  padding: 0;
+
+  & :hover:not(:active) {
+    background-color: inherit;
+    color: ${({ theme }) => theme.colors.primary70};
+  }
+
+  & :active {
```

Comment:
Do these need to not have spaces?
```suggestion
  &:hover:not(:active) {
    background-color: inherit;
    color: ${({ theme }) => theme.colors.primary70};
  }

  &:active {
```